### PR TITLE
new Thread configuration.

### PR DESCRIPTION
**Pull request by karsev**
Thread configuration needs to be updated (network, security). 
Also yotta defines need to be removed.

### CODE REVIEW

**Comment by karsev** [comment]
Thread network parameters to be updated

**Comment by karsev** [comment]
@artokin

File: configs/Thread_Atmel_RF_config.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            "mbed-trace.enable": 0,
+            "mbed-mesh-api.thread-config-panid": "0x0700",
+            "mbed-mesh-api.thread-config-network-name": "\"Thread Network\"",
+            "mbed-mesh-api.thread-config-commissioning-dataset-timestamp": "0x00000001",
+            "mbed-mesh-api.thread-config-extended-panid": "{0xf1, 0xb5, 0xa1, 0xb2,0xc4, 0xd5, 0xa1, 0xbd }",
+            "mbed-mesh-api.thread-master-key": "{0x10, 0x11, 0x22, 0x33, 0x44, 0x55, 0x66, 0x77, 0x88, 0x99, 0xaa, 0xbb, 0xcc, 0xdd, 0xee, 0xff}",
+            "mbed-mesh-api.thread-config-ml-prefix": "{0xfd, 0x0, 0x0d, 0xb8, 0x0, 0x0, 0x0, 0x0}",
+            "mbed-mesh-api.thread-config-pskc": "{0xc8, 0xa6, 0x2e, 0xae, 0xf3, 0x68, 0xf3, 0x46, 0xa9, 0x9e, 0x57, 0x85, 0x98, 0x9d, 0x1c, 0xd0}"
+        },
+        "NUCLEO_F401RE": {
```
Comment (artokin) [inline configs/Thread_Atmel_RF_config.json]: Would it be possible to wipe out configuration values that are not relevant for current config?

**Comment by artokin** [comment]
@SeppoTakalo, what kind of configuration nanostack ethernet should have?

+ @mikter

**Comment by JanneKiiskila** [comment]
rebuild

**Comment by yogpan01** [comment]
@karsev Do you still need this PR ? I suggest to rebase it and resolve merge conflict to get it merged.

**Comment by karsev** [comment]
This can be removed

**Pull request closed without merging**